app/src/App.tsx

```diff
@@ -253,8 +252,8 @@ function App() {
             element: (
```

Comment:
not a blocker, but we should clean this up to use one route for all task lists, add dynamic segments into the path, and then use useParams within TaskListContainer to pull the borough :
https://reactrouter.com/en/main/route/route#dynamic-segments

Reply:
cool, i can do that in a follow-up pr!